RELEASE_NOTES.md

```diff
@@ -1,3 +1,28 @@
+# Test Kitchen 2.0 Release Notes
+
+Test Kitchen 2.0 is primarily a code cleanup release, but as it does remove functionality that a limited number of users may have depended on we have chosen to release this as a new major version. We believe that nearly all users will experience no change with this new version, but it allowed us to remove a good amount of legacy logic that will make Test Kitchen a bit faster and certainly easier to maintain going forward.
```

Comment:
```suggestion
Test Kitchen 2.0 is primarily a code cleanup release, but as it does remove functionality upon a limited number of users may have depended, we have chosen to release this as a new major version. We believe that nearly all users will experience no change with this new version, but this release allows us to remove a good amount of legacy logic that will make Test Kitchen a bit faster and certainly easier to maintain going forward.
```

Reply:
Would `version` rather than `release` be a better word choice for the second sentence?